susemanager-utils/susemanager-sls/src/states/appstreams.py

```diff
@@ -0,0 +1,50 @@
+def change(name, appstreams_to_enable, appstreams_to_disable):
```

Comment:
Hmm, there are few things here I would consider:

1) Since this is an "state" module, the function names should not be imperative, but declarative. So, I would suggested maybe something like:

```suggestion
def configured(name, appstreams_to_enable, appstreams_to_disable):
```

So, an SLS can be written like:
```yaml
configure_appstreams_repos:
  appstream.configured:
    - enabled_appstreams: ["foobar"]
    - disabled_appstreams: ["deadbeef"]
```

2) You have to make sure this module can run in "test" mode, without doing any actual change on the system. More info: https://docs.saltproject.io/en/latest/ref/states/writing.html#test-state

3) Please, add some documentation as header for this module.

4) A `__virtual__` function would be nice here, to check the necessary execution module is available, like we do here: https://github.com/uyuni-project/uyuni/blob/f3163fc212e11d0c92cf79f0492923124d809464/susemanager-utils/susemanager-sls/src/modules/virt_utils.py#L30-L39